### ISSUE TITLE
Autorestart probably shouldn't restart a process if terminated intentionally

### ISSUE THREAD

**Issue by robknight** [pvolok/mprocs#143]
If a process is set to `autorestart: true` then it will restart when killed by mprocs, e.g. as a result of pressing 'x' or using the `term-proc` command.

I assume that the purpose of `autorestart` is to re-start processes if they crash, exit due to errors, or even due to user interaction in the terminal. However, I do not expect the process to restart if I have issued a command to terminate it from within mprocs. Is the current behaviour correct? If so, should there be a command which can terminate a process without auto-restarting?

**Comment by pvolok** [comment]
Yes, the process certainly should NOT autorestart if stopped manually. This is a bug.
